react/__tests__/Shipping.test.tsx

```diff
@@ -20,9 +20,9 @@ describe('Shipping (delivery)', () => {
     const order = twoDeliveries.orderGroup.orders[0]
     const delivery = order.deliveryParcels
 
-    const { getByText } = render(<Shipping deliveryPackages={delivery} />)
+    const { queryByText } = render(<Shipping deliveryPackages={delivery} />)
 
-    expect(getByText(/ - n˚ \d of \d/)).toBeDefined()
+    expect(queryByText(/ - n˚ \d of \d/)).toBeDefined()
```

Comment:
I'm not quite sure if this test would fail if 
`getBy<something>` and `queryBy<something>`do not return the same way when nothing is found. Take a look at this: https://testing-library.com/docs/react-testing-library/cheatsheet
If `queryByText` does not find anything, it returns `null`, and I'm pretty sure that `.toBeDefined()` would return `true`, so the test would always pass.
https://jestjs.io/docs/en/expect#tobedefined